docs-2.0/3.ngql-guide/8.clauses-and-options/limit.md

```diff
@@ -95,13 +95,9 @@ nebula> GO 3 STEPS FROM "player100" \
 nebula> GO 3 STEPS FROM "player102" \
         OVER * \
         LIMIT [rand32(5),rand32(5),rand32(5)];
-+------------+-------------+---------------------+
-| serve._dst | follow._dst | any_shape_edge._dst |
-+------------+-------------+---------------------+
-| "team204"  |             |                     |
-| "team215"  |             |                     |
-|            | "player100" |                     |
-+------------+-------------+---------------------+
++------------+-------------+
+| serve._dst | follow._dst |
++------------+-------------+
```

Comment:
@even1111112 Thanks for the PR!

The rand32(5) uses a random int as the seed, so the result changes from time to time, and sometimes it can be empty like the one you wrote. But it is recommended to show something for the reader to make the doc more meaningful.

You are right about deleting the `any_shape_edge._dst` column. It should not appear here.